.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build generator-symphony NodeJs project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs-or-python
+
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I guess we want to have a build on all PR on *-rc branches well

Reply:
Added in https://github.com/finos/generator-symphony/pull/169/commits/609037006fd50a1533c520960a3d37d58bd07301